styles/styles.css

```diff
@@ -318,6 +318,138 @@
   color: #383838;
 }
 
+.sign-up--mob {
+  display: none;
+}
+
+.destinations {
+  display: flex;
+  flex-direction: column;
+  padding: 120px var(--col-width);
+}
+
+.destinations--title {
+  margin: 0 auto;
+  padding-bottom: 104px;
+  font-weight: 500;
+  font-size: 40px;
+  line-height: 47px;
+
+  color: #383838;
+}
+
+.destinations--buttons {
+  width: 623px;
+  height: 64px;
+
+  display: flex;
+  background-color: #F3F3F4;
+  border-radius: 8px;
+}
+
+.destinations--button {
+  width: 33.33%;
+  height: 64px;
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+
+  color: #383838;
+  background-color: #F3F3F4;
+  border-radius: 8px;
+  border: 0;
+}
+
+.destinations--button:focus {
+  color: #FFFFFF;
+  background-color: #3077C6;;
+}
+
+.destinations--pictures {
```

Comment:
as you can see all you blocks have conten with three or four items, so you can create universal cals for item which will be have width 25% or in footer 33% and it will be wrapped with calss which will be have max-width property  like width on design and property width:100%. It are touched all your blocks
It will make your design more responsive

Reply:
don't forget about pseudoclasses like :first-child, :last-child, :nth-* and etc.

Reply:
also you can use col notation for your items

Reply:
please have a look now. i'm not sure i did now as you said.